ACMILabs/acmi-api
# #84 Add Creators endpoint

Resolves #84

Let's add `/creators` to the API.

### Acceptance Criteria
- [x] Add `/creators` to the API

### Relevant design files
* None

### Testing instructions
1. Start the development environment: `make up`
1. Import some creators (see below - let a couple of pages run - we may want to order the XOS Creators API before saving the JSON here)
1. Note Creators are on the home page: http://localhost:8081/
1. Visit the Creators endpoint: http://localhost:8081/creators/
1. Visit an individual Creator: http://localhost:8081/creators/86908/
1. Note there aren't any `image` fields on the API
1. Search Creators: http://localhost:8081/search/?query=pine&resource=creators

```python
$ docker exec -it api python

from app.api import Search, XOSAPI
xos = XOSAPI()
xos.get_creators()
search = Search()
search.update_index('creators')
```
